lib/runner.js

```diff
@@ -137,10 +161,11 @@ Runner.prototype._addTestResult = function (test) {
 	this.results.push({
 		duration: test.duration,
 		title: test.title,
-		error: test.assertError
+		error: test.assertError,
+		type: test.type
 	});
 
-	this.emit('test', test.assertError, test.title, test.duration);
+	this.emit('test', test.assertError, test.title, test.duration, test.type);
```

Comment:
Absolutely, was going to do that here at first, but then thought about a new PR :D